sdk/communication/tests.yml

```diff
@@ -52,4 +64,4 @@ stages:
       AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
       AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
       COMMUNICATION_PHONE_NUMBER: $(communication-livetest-phone-number-for-config)
-
+      TEST_PACKAGES_ENABLED: ${{ parameters.TestPackagesEnabled }}
```

Comment:
Is there a reason this has to be "${{" rather than "$(" as everything else in the surrounding code?